ui/lib/open-api-explorer/addon/components/swagger-ui.js

```diff
@@ -26,18 +26,24 @@ export default class SwaggerUiComponent extends Component {
     return {
       fn: {
         opsFilter: (taggedOps, phrase) => {
-          // map over the options and filter out operations where the path doesn't match what's typed
-          return (
-            taggedOps
-              .map((tagObj) => {
-                const operations = tagObj.operations.filter((operationObj) => {
-                  return operationObj.path.includes(phrase);
-                });
-                return tagObj.set('operations', operations);
-              })
-              // then traverse again and remove the top level item if there are no operations left after filtering
-              .filter((tagObj) => !!tagObj.operations.size)
-          );
+          const filteredOperations = taggedOps.reduce((acc, tagObj) => {
```

Comment:
the reduce function here allows us to combine the previous `map`, `filter` and  then`map` that we had before. 🎉